hibernate-core/src/test/java/org/hibernate/test/hql/HQLTest.java

```diff
@@ -241,10 +241,13 @@ public void testCollectionMemberDeclarations(){
 		// IN asks an alias, but the difference is that the error message from AST
 		// contains the error token location (by lines and columns), which is hardly 
 		// to get from Classic query translator --stliu
-		assertTranslation( "from Customer c, in(c.orders)" ); 
+		assertTranslation( "from Customer c, in(c.orders)" );
 	}
 
 	@Test
+	@FailureExpected(
```

Comment:
When introducing a configuration option, you can re-enable these tests.

Reply:
Imho these tests should never be deactivated, I'd rather see them adjusted to the new PR.

Reply:
Tests are re-enabled already.